changes/ocd-4373.sql

```diff
@@ -0,0 +1,323 @@
+
+
+call openchpl.backfill_standards('170.207(i)', 'Encounter diagnoses: The code set specified at 45 CFR 162.1002(c)(2) for the indicated conditions ICD-10-CM as maintained and distributed by HHS, for the following conditions:
+Diseases.
+Injuries.
+Impairments.
+Other health problems and their manifestations.
+Causes of injury, disease, impairment, or health problems.', 3, '2016-01-14', '2016-01-14', null, null, null, array[16, 19, 20, 21, 165]);
+
+
+call openchpl.backfill_standards('170.207(n)(1)', 'Birth sex must be coded in accordance with HL7 Version 3 Standard, Value Sets for AdministrativeGender and NullFlavor attributed as follows:
+Male. M; Female. F; Unknown. UNK', 3, '2016-01-14', '2016-01-14', '2025-12-31', 'Sex Code Set', 'Sex must be coded in accordance with, at a minimum, the version of SNOMED CT® codes specified in §170.207(a)(1)', array[5, 16, 28, 165]);
+
+
+call openchpl.backfill_standards('170.207(o)(1)', 'Sexual orientation must be coded in accordance with, at a minimum, the version of SNOMED CT® codes adopted at paragraph (a)(4) of this section for paragraphs (o)(1)(i) through (iii) of this section and HL7 Version 3 for paragraphs (o)(1)(iv) through (vi) of this section, attributed as follows:
+
+Lesbian, gay or homosexual. 38628009
+Straight or heterosexual. 20430005
+Bisexual. 42035005
+Something else, please describe. nullFlavor OTH
+Don’t know. nullFlavor UNK
+Choose not to disclose. nullFlavor ASKU', 3, '2016-01-14', '2016-01-14', '2025-12-31', 'SOGI', 'Sexual Orientation and Gender Identity must be coded in accordance with, at a minimum, the version of SNOMED CT® codes specified in §170.207(a)(1)', array[5]);
+
+call openchpl.backfill_standards('170.207(o)(2)', 'Gender identity must be coded in accordance with, at a minimum, the version of SNOMED CT® codes adopted at paragraph (a)(4) of this section for paragraphs (o)(2)(i) through (v) of this section and HL7 Version 3 for paragraphs (o)(2)(vi) and (vii) of this section, attributed as follows:
+
+Identifies as Male. 446151000124109
+Identifies as Female. 446141000124107
+Female-to-Male (FTM)/Transgender Male/Trans Man. 407377005
+Male-to-Female (MTF)/Transgender Female/Trans Woman. 407376001
+Genderqueer, neither exclusively male nor female. 446131000124102
+Additional gender category or other, please specify. nullFlavor OTH
+Choose not to disclose. nullFlavor ASKU', 3, '2016-01-14', '2016-01-14', '2025-12-31', 'SOGI', 'Sexual Orientation and Gender Identity must be coded in accordance with, at a minimum, the version of SNOMED CT® codes specified in §170.207(a)(1)', array[5]);
+
+
+call openchpl.backfill_standards('170.210(e)(1)', 'The audit log must record the information specified in sections 7.1.1 and and 7.1.2 and 7.1.6 through 7.1.9 of the standard specified in § 170.210(h) and changes to user privileges when health IT is in use.
```

Comment:
Is criteria ID 174 left out on purpose? It's in the spreadsheet Row 139 but it's yellow and I don't know what yellow means

Reply:
Ok, as long as there is a plan to follow up on it. Might be good to let QA note that so they don't find the same missing criteria.